packages/labs/observers/scripts/dev.sh

```diff
@@ -0,0 +1,25 @@
+#!/usr/bin/env bash
```

Comment:
These have been copied over for a while, but after talking to Al I don't think anyone's using them. Are you? If not, I'd start removing them...

Reply:
Done.